addon/components/file-chooser/component.js

```diff
@@ -17,10 +20,12 @@ import layout from './template';
  * Exposed to block context
  *  - `this`: the component object itself, so the block can invoke actions
  *      example:
+ *      ```javascript
```

Comment:
Good point- took another look at the ember docs, and there may even be support for handlebars highlighting mode. I'll go through and make these changes.